projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/trait-wpcom-rest-api-proxy-request-trait.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Trait WPCOM_REST_API_Proxy_Request_Trait
+ *
+ * Used to proxy requests to wpcom servers.
+ *
+ * @package automattic/jetpack
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Status\Visitor;
+
+trait WPCOM_REST_API_Proxy_Request_Trait {
+
+	/**
+	 * Proxy request to wpcom servers for the site and user.
+	 *
+	 * @param WP_Rest_Request $request Request to proxy.
+	 * @param string          $path Path to append to the rest base.
+	 *
+	 * @return mixed|WP_Error           Response from wpcom servers or an error.
+	 */
+	public function proxy_request_to_wpcom( $request, $path = '' ) {
```

Comment:
How come nothing like this wasn't already in place? What's unique that we're doing here?

Reply:
there are at least 4 other endpoints that have a similar implementations already, I wrote this one as generic as possible but I don't want to refactor all the endpoints in this PR.